packages/core/useRafFn/index.ts

```diff
@@ -27,13 +28,14 @@ export function useRafFn(fn: Fn, options: RafFnOptions = {}): Pausable {
   } = options
 
   const isActive = ref(false)
+  const rafId = ref(null) as Ref<null | number>
```

Comment:
```suggestion
  const rafId: number | undefined
```

This does not need to be a ref